packages/salesforcedx-sobjects-faux-generator/test/fauxgeneratortest.ts

```diff
@@ -176,4 +238,105 @@ describe('generate relationship tests', function() {
     expect(classText).to.include('Account Foo');
     expect(classText).to.include('List<Account> Reference');
   });
+
+  it('faux class generator should handle external lookup relationship', async function(): Promise<
+    void
+  > {
+    const field1 =
+      '{"name": "ExtRef__c", "type": "reference", "referenceTo": [], "relationshipName": null, "extraTypeInfo": "externallookup"}';
+    const header = '{ "name": "Custom__c",  "childRelationships": []';
+    const fieldHeader = '"fields": [';
+    const sobject1 = `${header},${fieldHeader}${field1}]}`;
+    const sobjectFolder = './';
+    const gen: FauxClassGenerator = new FauxClassGenerator();
+    classPath = await gen.generateFauxClass(
+      sobjectFolder,
+      JSON.parse(sobject1)
+    );
+    expect(fs.existsSync(classPath));
+    const classText = fs.readFileSync(classPath, 'utf8');
+    expect(classText).to.include('String ExtRef__c');
+  });
+
+  // Note - hierarchical relationship, many-to-many, and master-detail all look like lookup relationships in terms of
+  //  the relevant parts of describe
+  // For some of these, can create another test with extra info that is present, but it isn't paid attention to
+  // Might be good for completeness in case this other info eventually is relevant, but will  not be addressed currently
+});
+
+// Note, currently __x (ExternalObject) is not handled by describe (REST or Cli), but is handled by SDD
```

Comment:
I'd just take this out. If externalObject isn't an sobject in terms of list/describe, we don't need to worry about it.